lib/src/emoji_lists.dart

```diff
@@ -739,6 +739,7 @@ final Map<String, String> animals = Map.fromIterables([
   'Fox Face',
   'Bear Face',
   'Panda Face',
+  'Koala',
```

Comment:
Ah yeah, "Koala Face" seems to be the most correct term here 🙇 Please change! 🙏

Reply:
✅